### PR TITLE
F429 discovery ram dev

### PR DESCRIPTION
**Pull request by dron0gus**
Temporally PR for build testing.

### CODE REVIEW

**Comment by dron0gus** [comment]
Screenshot from 2022-01-09 18-10-46

File: firmware/config/stm32f4ems/efifeatures.h [hunk 1]
```diff
@@ -257,10 +257,14 @@
 
 #define EFI_CONSOLE_USB_DEVICE SDU1
 
-// F42x has more memory, so we can:
-//  - use compressed USB MSD image (requires 32k of memory)
-//  - use perf trace (requires ~16k of memory)
-#ifdef EFI_IS_F42x
+#if defined(EFI_HAS_EXT_SDRAM)
+    #define ENABLE_PERF_TRACE TRUE
+    #define LUA_USER_HEAP (1 * 1024 * 1024)
```
Comment (rusefillc) [inline firmware/config/stm32f4ems/efifeatures.h:262]: WOW that's so cool!

Comment (dron0gus) [reply firmware/config/stm32f4ems/efifeatures.h:262]: Actually there is 8Mb. You can allocate whole SDRAM to LUA.

Comment (rusefillc) [reply firmware/config/stm32f4ems/efifeatures.h:262]: nope, let's keep this as is with 1Mb for now :)

**Pull request merged**